src/utils/shim.js

```diff
@@ -1,21 +1,3 @@
-/*
```

Comment:
Are you sure we don't need this here?

Reply:
@jsidorenko because this file is autogenerated it removed the `Copyright notice` ... I'm updating the changes to include this `notice`